libs/ledger-live-common/src/featureFlags/defaultFeatures.ts

```diff
@@ -463,6 +463,12 @@ export const DEFAULT_FEATURES: Features = {
       threshold: 75,
     },
   },
+  marketperformanceWidgetDesktop: {
+    enabled: false,
+    params: {
+      variant: ABTestingVariants.variantA,
+    },
+  },
```

Comment:
Is there a chance the FF gets activated in the backend making screenshots unreliable ?

Reply:
Nope normally it's based on this file